src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
+
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    static final String WHITE_SPACE = " ";
+    static final int DATE_INDEX = 0;
+    static final int NAME_INDEX = 1;
+    static final int HOURS_INDEX = 2;
+    static final int SALARY_INDEX = 3;
```

Comment:
If we use constants only within the class, then we create them with access type private